core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.html

```diff
@@ -9,6 +9,7 @@
     <span class="oppia-ck-editor-copy-toolbar-tip-sublabel">
       Use the copy tool to easily copy content like images
     </span>
+    <p>(Press <span class="key-style">ALT + 0</span> to see accessibility menu for Rich text editor (RTE))</p>
```

Comment:
Do we have any mock for the UI/UX for this bar (suggested/created by UI/UX team)? if not, I would suggest asking UI/UX team members to create one.

I have the following suggestions/concern:
- I'm not sure whether "RTE" is a popular term and can be understood by translators, should we use a different term? 
- Do we need to keep this in brackets `()`? (Maybe an "i" icon and hover should show a pop-up (not sure!))
- This is written just below the "Copy tool" and seems like related to copy-tool, maybe a better placement?